data/Language/JavaScript.yaml

```diff
@@ -19,3 +19,11 @@ delimiters:
   - round_braces
   - square_braces
   - backticks
+datatypes:
+  - boolean_js
```

Comment:
This can be `boolean_toml` , as the JS datatype boolean is the same as the toml boolean: it has two literals, and doesnt have a keyword.